tests/ArrayHelper/RemoveTest.php

```diff
@@ -95,6 +95,16 @@ public function testRemoveByFloatKey(): void
         $this->assertSame([1 => 'a', 3 => 'c'], $array);
     }
 
+    public function testRemoveByFloatKey2(): void
```

Comment:
It's better to add tests to the method above than add a new method.

Reply:
Done!